examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model = model = mm.DLRMModel(
</pre><p> </p><p>I think there is a cut-n-paste error here. Instead of <code>model = model = ...</code> we just go <code>model = model</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
thanks